src/service/BookingService.java

```diff
@@ -0,0 +1,37 @@
+package service;
+
+import java.math.BigDecimal;
+import java.time.Duration;
+import java.time.LocalDateTime;
+
+import database.dao.BookingDataDAO;
+import database.dto.BookingDataDTO;
+import database.dto.PaymentMethodDTO;
+
+public class BookingService {
+	
+	private final BookingDataDAO bookingDataDAO;
+	private final BigDecimal pricePerNight=new BigDecimal("5000");
```

Comment:
This price must be configurable, not all the hotels will charge 5000 units per night, please review the `db_url` configuration variable and do the same for `pricePerNight`.